stix_shifter_modules/reaqta/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -1824,8 +1824,8 @@
         "object": "x-reaqta"
       },
       "tactics": {
-        "key": "x-reaqta-event.tactics",
-        "object": "x-reaqta"
+        "key": "x-ibm-finding.ttp_tagging_refs",
```

Comment:
You need to also create an `x-ibm-ttp-tagging` object when the tactics and technique fields are in the native results. This is the object that the x`-ibm-finding.ttp_tagging_refs` will reference. For an example of referencing, take a look at how the `x-oca-asset.ip_refs` references the `src_ip` in this same file.